lib/MentionsForm/MentionsFormInput.js

```diff
@@ -33,8 +37,20 @@ class MentionsFormInput extends Component {
 
   componentDidMount() {
     this.setState({ inputHeight: this.getInputHeight() }); // eslint-disable-line react/no-did-mount-set-state
+    this.addDefaultUsers();
   }
 
+  addDefaultUsers = async () => {
+    const { defaultUsers } = this.props;
+    if (defaultUsers.length) {
+      defaultUsers.map(user =>
+        setTimeout(() => {
```

Comment:
Open to other suggestions to get around this... 

I came across a bit of a strange race condition with this:
Component would mount -> map through and set our default users -> UpdateInputValue triggers -> map finishes before state can ever update here https://github.com/gathercontent/gather-ui/pull/485/files#diff-4d3f51f4b10ea07b6defa87007b6bc22R90 -> each default user overrides the previous because the input value hasnt yet updated from the previous default users.

Reply:
This is a really tough one!

The only thing I can think of is to see whether the `react-mentions` has the option to add multiple mentions in one action. This way we would only update the input and the state once.

Either that, or `react-mention` may have the option to set the default users?

The other option would be to make the input uncontrolled, but that would be a MASSIVE refactor.